<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>719</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>731</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>815</width>
    <height>731</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Log In</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(16,30,41,240);
color:rgba(255,255,255,150);
border-radius:30px;
border-color:rgba(255,255,255,255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>26</width>
    <height>26</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1251</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255,255,255,200);
font: 42pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>WATCH</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>431</width>
        <height>731</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-radius:10px;
background-color:qlineargradient(spread:pad, x1:0.858, y1:0.375, x2:0, y2:0, stop:0 rgba(16, 30, 41, 200), stop:1 rgba(0, 1, 145, 255));
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>1251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>71</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 42pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;
color: rgba(35,125,236,255);</string>
      </property>
      <property name="text">
       <string>VE</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>811</width>
        <height>651</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>811</width>
        <height>631</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>242424</width>
        <height>242424</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>811</width>
        <height>631</height>
       </size>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(16,30,41,240);
color:rgba(255,255,255,150);
border-radius:10px;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_Username_11">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>521</width>
         <height>51</height>
        </rect>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>250</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px;
border-radius:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>User Name</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Password_11">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>340</y>
         <width>521</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px;
border-radius:0px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ok_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>450</y>
         <width>521</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#ok_13{
	font: 75 8pt &quot;Poppins&quot;;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(2, 65, 118,255);
color: rgba(255, 255, 255,200);
border-radius:8px;
}
QPushButton#ok_13:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(2, 65, 118,100);
background-position:calc(100% - 10px)center;
}
QPushButton#ok_13{
background-color: rgba(2, 65, 118,200);
}
QPushButton#ok_13:hover{
background-color: rgba(0,125,236,255);
}
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>271</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 90pt &quot;MS Shell Dlg 2&quot;;
color:rgba(0,125,236,255);</string>
       </property>
       <property name="text">
        <string>ÓÅñ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>550</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(255,255,255,150);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Forgot your Username or Password ?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ku_12">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>550</y>
         <width>75</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#ku_12{
background-color:transparent;
color: rgba(0,125,236,255);
}
QPushButton#ku_12:hover{
background-color:transparent;
color: rgba(50,120,236,255);
text-decoration:underline;
}</string>
       </property>
       <property name="text">
        <string>Click Here</string>
       </property>
      </widget>
      <widget class="QPushButton" name="twitter_11">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>590</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#twitter_11{
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(50,120,236,255);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
background-color:rgba(16,30,41,240);
}
QPushButton#twitter_11:hover{
background-color: rgba(50,120,236,200);
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(16,30,41,240);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="face_11">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>590</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#face_11{
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(50,120,236,255);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
background-color:rgba(16,30,41,240);
}
QPushButton#face_11:hover{
background-color: rgba(50,120,236,200);
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(16,30,41,240);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
      <widget class="QPushButton" name="youtube_11">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>590</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#youtube_11{
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(50,120,236,255);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
background-color:rgba(16,30,41,240);
}
QPushButton#youtube_11:hover{
background-color: rgba(50,120,236,200);
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(16,30,41,240);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
      <widget class="QPushButton" name="insta_11">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>590</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#insta_11{
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(50,120,236,255);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
background-color:rgba(16,30,41,240);
}
QPushButton#insta_11:hover{
background-color: rgba(50,120,236,200);
font: 24pt &quot;Social Media Circled&quot;;
color: rgba(16,30,41,240);
border-color:rgba(50,120,236,255);
border:2px solid rgba(50,120,236,255) ;
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Q</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>590</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,125,236,255);
font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Find us on :</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="ok_14">
      <property name="geometry">
       <rect>
        <x>1160</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#ok_14{
background-color:rgba(0,125,236,255);
color: rgb(0, 0, 0);
font: 15pt &quot;Permanent Marker&quot;;
border-radius:10px;
}
QPushButton#ok_14:hover{
background-color:rgba(236,125,0,255);
color: rgb(255,255, 255);
font: 15pt &quot;Permanent Marker&quot;;
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ku_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>380</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#ku_13{
background-color:transparent;
color: rgba(0,125,236,255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton#ku_13:hover{
background-color:transparent;
color: rgba(50,120,236,255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
text-decoration:underline;
}</string>
      </property>
      <property name="text">
       <string>Forgot password...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ok_15">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#ok_15{
background-color:rgba(0,125,236,255);
color: rgb(0, 0, 0);
font: 10pt &quot;Permanent Marker&quot;;
border-radius:10px;
}
QPushButton#ok_15:hover{
background-color:rgba(236,125,0,255);
color: rgb(255,255, 255);
font: 10pt &quot;Permanent Marker&quot;;
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>391</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255,255,255,200);
font: 42pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Welcome ,</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255,255,255,200);
font: 24pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Already have an account ?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_gif_animation">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>-20</y>
        <width>1271</width>
        <height>751</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_gif_animation</zorder>
     <zorder>label_28</zorder>
     <zorder>widget_2</zorder>
     <zorder>ok_15</zorder>
     <zorder>label_34</zorder>
     <zorder>ku_13</zorder>
     <zorder>label_27</zorder>
     <zorder>label_29</zorder>
     <zorder>label_30</zorder>
     <zorder>label_33</zorder>
     <zorder>ok_14</zorder>
    </widget>
    <widget class="QWidget" name="forogtpaasword">
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>841</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255,255,255,200);
font: 40pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Please Enter Your Email Address :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>731</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>250</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0);
font: 18pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px;
border-radius:0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email Address</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::VisualMoveStyle</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="ok_16">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>510</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#ok_16{
	font: 75 8pt &quot;Poppins&quot;;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(2, 65, 118,255);
color: rgba(255, 255, 255,200);
border-radius:8px;
}
QPushButton#ok_16:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(2, 65, 118,100);
background-position:calc(100% - 10px)center;
}
QPushButton#ok_16{
background-color: rgba(2, 65, 118,200);
}
QPushButton#ok_16:hover{
background-color: rgba(0,125,236,255);
}
</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgba(255,255,255,200);
font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>(Note*) You Will Receive A Mail After A Short Period Of Time That Has Your Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="server">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>189</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>465</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uname">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>souhaib.touaiti@esprit.tn</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paswd">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>249</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>221JMT5189</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>201</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="plainText">
       <string>7865</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
